lms/migrations/versions/7858a12c4e4a_add_index_for_user_deduplication.py

```diff
@@ -0,0 +1,24 @@
+"""Add index for user deduplication."""
+
+import sqlalchemy as sa
+from alembic import op
+
+revision = "7858a12c4e4a"
+down_revision = "f6c442c861c4"
+
+
+def upgrade() -> None:
+    # CONCURRENTLY can't be used inside a transaction. Finish the current one.
+    op.execute("COMMIT")
+
+    op.create_index(
+        "ix__user_h_user__updated",
```

Comment:
```suggestion
        "ix__user__h_userid__updated",
```

Reply:
I think we are erring on the side of consistency versus choosing a better name.

Updated it to match the downgrade migration :+1:

Reply:
> I think we are erring on the side of consistency versus choosing a better name.

Better in what way? We can deviate from existing practices if we have a good reason, but doing so in a haphazard way makes it difficult for other developers to work with the code in future.